.github/workflows/test-upstream.yml

```
@@ -42,7 +42,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         id: rust-toolchain
         with:
-          toolchain: stable
+          toolchain: nightly
```

Does this still need to be nightly, or can we switch back to the stable toolchain?

```suggestion
          toolchain: stable
```